buildbot/configure.py

```diff
@@ -25,6 +25,7 @@ def do_configure(args):
     llvm_enable_projects = 'clang;' + llvm_external_projects
     libclc_targets_to_build = ''
     sycl_build_pi_cuda = 'OFF'
+    sycl_build_pi_cuda = 'OFF'
```

Comment:
```suggestion
    sycl_build_pi_rocm = 'OFF'
```
?

Reply:
My mistake, should be `sycl_build_rocm` .